src/sagemaker/workflow/pipeline.py

```diff
@@ -828,34 +880,17 @@ def list_steps(self):
         """Describes a pipeline execution's steps.
```

Comment:
To provide consistent user experience among list_pipelines, list_executions, and list_steps, we should define `_StepList` class as well, which extends the `_ListBase` class and have a `get_table`/`display_table` method to get the pandas table.

let's add a TODO for the change